src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -158,22 +158,18 @@ def two_groups_hyphens_two_replacements_with_suffix(
     "opencv-python-headless": ("cv2",),
     "opensearch-py": ("opensearchpy",),
     # opentelemetry
-    "opentelemetry-api": ("opentelemetry",),
```

Comment:
This one is bogus. `opentelemetry` is a namespace. The list of modules attributed to `opentelemetry-sdk` actually belongs to `opentelemetry-api`, according to the docs linked to from #20551 (so was a mistake to begin with.)

Reply:
My bad, messed that up